### DISCUSSION TITLE
Welcome to Arc Mini Discussions!

### DISCUSSION THREAD

**Discussion by sobri909** [sobri909/ArcMini#115] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikhailsdv** [comment]
👋 Hi. I’ve been using the ArcMini app for almost three years, and I’ve always wanted the ability to select date ranges for GPX/JSON exports. Currently, I can only export data for a single day. It would be very useful to export data for longer periods, such as months or years. Are you planning to add this feature anytime soon?

**Reply by sobri909** [reply]
Hi! Arc Mini hasn't been maintained for many years. The supported apps are Arc Timeline and Arc Editor. I recommend checking the support forum: https://support.bigpaua.com
